core/src/action.rs

```diff
@@ -92,6 +92,7 @@ impl Hash for ActionWrapper {
 /// All Actions for the Holochain Instance Store, according to Redux pattern.
 #[derive(Clone, PartialEq, Debug, Serialize)]
 #[serde(tag = "action_type", content = "data")]
+#[allow(clippy::large_enum_variant)]
```

Comment:
There might be some truth here about the size of our enums and I think we could make them a bit smaller? Some of them already look like they are organized by sections :)